src/dlr_relayvm.cc

```diff
@@ -0,0 +1,283 @@
+#include "dlr_relayvm.h"
+
+
+#include <stdlib.h>
+#include <fstream>
+#include <iterator>
+#include <numeric>
+
+using namespace dlr;
+
+const std::string RelayVMModel::ENTRY_FUNCTION = "main";
+
+void RelayVMModel::InitModelPath(std::vector<std::string> paths) {
+  path_ = std::make_unique<ModelPath>();
+  std::vector<std::string> paths_vec;
+  for (auto path : paths) {
+    ListDir(path, paths_vec);
```

Comment:
It's better to reactor ListDir to take the pointer and pass &path_vec here, so it is more explicit that path_vec will be updated.

Reply:
Yes https://github.com/neo-ai/neo-ai-dlr/pull/213/files#diff-092ae707d8211b15c457e61d3f496d85R49